Kudu.Contracts/Scan/ScanReport.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Kudu.Contracts.Infrastructure;
+using Newtonsoft.Json;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Kudu.Contracts.Scan
+{
+    public class ScanReport : INamedObject
```

Comment:
Do we need to inherit `INamedObject` ?

Reply:
I have used the ArmUtils.AddEnvelopeOnArmRequest() on the result which takes in instances of INamedObject. I saw this in the DeploymentController. Will we need this when we implement the REST API calls from the controller?